### PR TITLE
[PORT] port crm_claim_categ_as_name to v 7.0

### PR DESCRIPTION
**Pull request by bguillot**
Port crm_claim_categ_as_name to v 7.0

### CODE REVIEW

**Comment by bguillot** [comment]
Travis failed because of the other modules

**Comment by rdeheele** [comment]
:+1:

File: crm_claim_categ_as_name/__init__.py [hunk 1]
```diff
@@ -1,8 +1,8 @@
-# -*- encoding: utf-8 -*-
+# -*- coding: utf-8 -*-
 ###############################################################################
 #                                                                             #
 #   crm_claim_categ_as_name for OpenERP                                       #
-#   Copyright (C) 2012 Akretion Benoît GUILLOT <benoit.guillot@akretion.com>  #
+#   Copyright (C) 2014 Akretion Benoît GUILLOT <benoit.guillot@akretion.com>  #
```
Comment (pedrobaeza) [inline crm_claim_categ_as_name/__init__.py]: Please respect full log: 2012-2014

**Comment by bguillot** [comment]
Thanks for the reviews, I changed the copyrights

**Comment by pedrobaeza** [comment]
Thanks :+1:

File: crm_claim_categ_as_name/claim_view.xml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!--
   crm_claim_categ_as_name for OpenERP
-  Copyright (C) 2012 Akretion Benoît GUILLOT <benoit.guillot@akretion.com>
+  Copyright (C) 2014 Akretion Benoît GUILLOT <benoit.guillot@akretion.com>
```
Comment (yvaucher) [inline crm_claim_categ_as_name/claim_view.xml]: Also here please restore 2012-2014

**Comment by yvaucher** [comment]
Just a last one to fix and we are good.

Thanks

**Comment by bguillot** [comment]
Hum that's right my bad ...
Thank you for the review

**Comment by yvaucher** [comment]
:+1:

**Pull request merged**